src/entities/conditional-logic/model/conditions.test.ts

```diff
@@ -0,0 +1,327 @@
+    { input: 19, min: 20, max: 30 },
+    { input: 31, min: 20, max: 30 },
+    { input: 20, min: 20, max: 30 },
+    { input: 30, min: 20, max: 30 },
+    { input: 0, min: 1, max: 3 },
+  ];
+
+  outsideTestCases.forEach(({ input, min, max }) => {
+    it(`should return "false" for the input: ${input} that is outside of values range: [${min}, ${max}]`, () => {
+      expect(isBetweenValues(input, min, max)).toBe(false);
+    });
+  });
+
+  const borderLineValuesTestCases = [
+    { input: 20, min: 20, max: 30 },
+    { input: 30, min: 20, max: 30 },
+    { input: 0, min: 0, max: 5 },
+  ];
+
+  borderLineValuesTestCases.forEach(({ input, min, max }) => {
+    it(`should return "false" for the borderline value ${input} of the values range: [${min}, ${max}]`, () => {
+      expect(isOutsideOfValues(input, min, max)).toBe(false);
+    });
+  });
+
+  const nilTestCases = [
+    { input: null, min: 20, max: 30 },
+    { input: undefined, min: 20, max: 30 },
+  ];
+
+  nilTestCases.forEach(({ input, min, max }) => {
+    it(`should return "false" when input is ${input}`, () => {
+      expect(isBetweenValues(input, min, max)).toBe(false);
+    });
+  });
+});
+
+describe('[isOutsideOfValues] function', () => {
+  const outsideTestCases = [
+    { input: 19, min: 20, max: 30 },
+    { input: 31, min: 20, max: 30 },
+    { input: 0, min: 2, max: 10 },
+  ];
+
+  outsideTestCases.forEach(({ input, min, max }) => {
+    it(`should return "true" for the input: ${input} that is outside of the values range: [${min}, ${max}]`, () => {
+      expect(isOutsideOfValues(input, min, max)).toBe(true);
+    });
+  });
+
+  const inRangeTestCases = [
+    { input: 25, min: 20, max: 30 },
+    { input: 1, min: 0, max: 2 },
+    { input: 0, min: -2, max: 2 },
+  ];
+
+  inRangeTestCases.forEach(({ input, min, max }) => {
+    it(`should return "false" when the input ${input} is between the values range: [${min}, ${max}]`, () => {
+      expect(isOutsideOfValues(input, min, max)).toBe(false);
+    });
+  });
+
+  const borderLineValuesTestCases = [
+    { input: 20, min: 20, max: 30 },
+    { input: 30, min: 20, max: 30 },
+    { input: 0, min: 0, max: 30 },
+  ];
+
+  borderLineValuesTestCases.forEach(({ input, min, max }) => {
+    it(`should return "false" for the borderline value ${input} of the values range: [${min}, ${max}]`, () => {
+      expect(isOutsideOfValues(input, min, max)).toBe(false);
+    });
+  });
+
+  const nilTestCases = [
+    { input: null, min: 20, max: 30 },
+    { input: undefined, min: 20, max: 30 },
+  ];
+
+  nilTestCases.forEach(({ input, min, max }) => {
+    it(`should return "false" when input is ${input}`, () => {
+      expect(isOutsideOfValues(input, min, max)).toBe(false);
+    });
+  });
+});
+
+describe('[isEqualToValue] function', () => {
+  const emptyObject = {};
+  const sameValueTestCases = [
+    { a: 1, b: 1 },
+    { a: 'Hello', b: 'Hello' },
+    { a: 0, b: 0 },
+    { a: emptyObject, b: emptyObject },
```

Comment:
Are you testing equality by reference here? If so, then do not emphasize that object is empty here.

Reply:
Yes, you're right. Will change it